lib/renderer/content-scripts-injector.js

```diff
@@ -23,12 +23,30 @@ const runContentScript = function (extensionId, url, code) {
   return compiledWrapper.call(this, context.chrome)
 }
 
+const runStylesheet = function (url, code) {
+  const wrapper = `(function (code) {
+    function init() {
+      var styleElement = document.createElement('style');
+      styleElement.setAttribute('type', 'text/css');
+      styleElement.textContent = code;
+      document.head.append(styleElement);
+    }
+    document.addEventListener('DOMContentLoaded', init);
+  })`;
+  const compiledWrapper = runInThisContext(wrapper, {
+    filename: url,
+    lineOffset: 1,
+    displayErrors: true,
+  });
+  return compiledWrapper.call(this, code);
+}
+
 // Run injected scripts.
 // https://developer.chrome.com/extensions/content_scripts
 const injectContentScript = function (extensionId, script) {
   if (!script.matches.some(matchesPattern)) return
 
-  if (script.js) {
+  if (script.js.length) {
```

Comment:
We want this to be `script.js`, if the length is 0 we don't care, the loop won't execute.

We need to check if the `js` property is defined at all